src/index.html

```diff
@@ -15,13 +15,391 @@
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          class="moyo"
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__items">
+            <a
+              href="index.html"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__items">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div>
+          <img
+            class="card__product"
+            src="/src/images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+          />
+        </div>
+
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+        <div class="stars">
+          <div class="stars__container">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-price">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          class="card__botton"
+          href="#"
+          data-qa="card-hover"
+        >
+          <strong>Buy</strong>
```

Comment:
you actually don't need to wrap this text in strong tag 
You can add styles via css